src/lib/core/a11y/interactivity-checker.ts

```diff
@@ -16,6 +17,8 @@ import {Injectable} from '@angular/core';
 @Injectable()
 export class InteractivityChecker {
 
+  constructor(private platform: MdPlatform) {}
+
   /** Gets whether an element is disabled. */
   isDisabled(element: HTMLElement) {
```

Comment:
Would rather address things like that in another PR. We should keep that one as small as possible.